docs/source/installation.rst

```diff
@@ -84,7 +99,7 @@ Step 2: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
 
 ::
 
-    git clone git@github.com:ivadomed/ivadomed.git
+    git clone https://github.com/neuropoly/ivadomed.git
```

Comment:
no more on the neuropoly org-- ivadomed now has its own org

Reply:
Ah yes! My mistake I was the one to suggest this URL. The correct one would be `https://github.com/ivadomed/ivadomed.git` instead.
Interestingly, `https://github.com/neuropoly/ivadomed.git` redirects you to the right page.

Reply:
> Interestingly, https://github.com/neuropoly/ivadomed.git redirects you to the right page.

that's because we did a transfer between organizations (and GH sets the redirection automatically)

Reply:
Long pending both this has been fixed!